Makefile

```diff
@@ -27,6 +27,10 @@ ifneq ($(USE_PTRGUARD),)
 	BUILD_TAGS := $(BUILD_TAGS),ptrguard
 endif
 
+ifneq ($(USE_CACHE),)
+	GOCACHE_VOLUME := -v test_ceph_go_cache:/go
```

Comment:
1. Does this work without creating the directory in advance?
2. Should `make clean` delete the directory`?
3. Should the directory be added to `.gitignore`?
4. Does it not need `$(VOLUME_FLAGS)`?

Reply:
I guess there's a misunderstanding. `test_ceph_go_cache` is not a directory but the name of a named volume. It only exists inside of the container runtime. So the only point that applies would be 2), clean could make a `docker volume rm test_ceph_go_cache`. But I'm not sure, that's what would be expected by people, that a `make clean` fiddles with the CRE.